infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryServiceAdapter.java

```diff
@@ -9,22 +9,39 @@
 public interface CorfuReplicationDiscoveryServiceAdapter {
 
     /**
-     *
+     * update the topology configuration
      * @param topologyConfiguration
      */
     void updateTopology(LogReplicationClusterInfo.TopologyConfigurationMsg topologyConfiguration);
 
     /**
-     *
-     * @return
+     * Retrieves replication status from LR metadata tables
+     * @return Map of remoteClusterID to replicationStatus
      */
     Map<String, LogReplicationMetadata.ReplicationStatusVal> queryReplicationStatus();
 
     /**
      * Enforce snapshotFullSync
+     * @return force snapshot sync ID
      */
     UUID forceSnapshotSync(String clusterId) throws LogReplicationDiscoveryServiceException;
 
-
+    /**
+     * Get the role of local cluster if appropriate
```

Comment:
Is Role type and role (name) same?

Reply:
Yes.

JFYI, this file is getting removed in the sessionManager PR.